scripts/benchmark_parse.py

```
@@ -0,0 +1,19 @@
+import time
+
+from parse_grid import parse_grid_html
+
+
+def benchmark_parse():
+    with open("grid_05-08-2025.html", "r") as f:
```

Ruff (0.15.0)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)